### ISSUE TITLE
[Bug]: repeatable jobs not working with python

### ISSUE THREAD

**Issue by jaredmcqueen** [taskforcesh/bullmq#2772]
### Version

5.13.0

### Platform

NodeJS

### What happened?

according to the docs:

```
Every time a repeatable job is picked up for processing, the next repeatable job is added to the queue with a proper delay. Repeatable jobs are thus nothing more than delayed jobs that are added to the queue according to some settings.
```

I have tried both cron schedule `pattern` and `every`.  Both create the repeated job scaffolding in redis, and both jobs work _on the first run_

after the first job run, no new job is rescheduled, so jobs only ever repeat once.  

<img width="1626" alt="image" src="https://github.com/user-attachments/assets/b0c485c8-0545-42f8-94fa-8766d9578d11">

<img width="1625" alt="image" src="https://github.com/user-attachments/assets/361b42ce-81e2-4878-9d1f-38f3fd9eee6b">

### How to reproduce.

create a repeatable job in typescript

pull the jobs in python

`jobId` does look different with repeatable jobs - like this:
`jobId='repeat:dec4ba3039956b6560feae8ac0a67850:1726582010000`

### Relevant log output

```shell
N/A
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by roggervalf** [comment]
hi @jaredmcqueen, repeatable jobs are not currently supported in python version. Still working on the next breaking change for node version, so probably after that we can revisit this feature request

**Comment by jaredmcqueen** [comment]
@roggervalf thanks for the update.  What kind of timeline do you have in mind?  Huge fan of this project and would be willing to help speed it along any way I can.  Let me know what I can do to help.

**Comment by roggervalf** [comment]
hope that this week we can merge the next breaking change. Then we need to update pro versión and after that we will have time to revisit new feature request. So probably in 3 o 4 weeks.

**Comment by jaredmcqueen** [comment]
hey @roggervalf just checking in after a few weeks.  Was thinking about writing a workaround for python / cron.  Holler if you have a timeline for true implementation.  Again, I can help where needed

**Comment by jaredmcqueen** [comment]
hey @roggervalf checking in to see if there's any update on the python client grabbing repeatable jobs.  I know you're working hard releasing pro features and working to stabilize the TS/JS package.  Thanks

**Comment by roggervalf** [comment]
Hi @jaredmcqueen sorry for the delay. Something that is needed is a similar package as https://www.npmjs.com/package/cron-parser to be used in python. Then logic can be backported. I'm still working on job scheduler logic these days. I was thinking on porting this logic after next breaking change, but we are also opened for pull request if someone wants to help on this feature.

**Comment by jaredmcqueen** [comment]
yeah it seems that https://github.com/agronholm/apscheduler seems to be the go-to package for cron scheduling in python.  supports async too which is nice

**Comment by bforbanks** [comment]
would be nice for the scheduler to work in python 🙏

**Comment by andnig** [comment]
I know this is a super-hack, but I had a similar setup, where nodejs is creating the jobs and python should consume them.
Due to missing repeated (job-scheduler) support in python, I was forced to implement a workaround, which I might share.
And just to be frank here, this is a super-hack, but works for me 😅 :

I have implemented a separate worker in nodejs, which listens to a specific "repeated" job queue. This worker runs every x minutes and all it does is to check if a sync is already running (I have a threadsafe lock in my database for that), and if not, it creates a "normal" bullmq job, which the python service is listening to.

This, at least for my use case, mimics repeated job behavior. Maybe this helps some folks coming here.

I'm happy though, when repeated jobs arrive in python 👍

**Comment by Uziniii** [comment]
> I know this is a super-hack, but I had a similar setup, where nodejs is creating the jobs and python should consume them.
> Due to missing repeated (job-scheduler) support in python, I was forced to implement a workaround, which I might share.
> And just to be frank here, this is a super-hack, but works for me 😅 :
> 
> I have implemented a separate worker in nodejs, which listens to a specific "repeated" job queue. This worker runs every x minutes and all it does is to check if a sync is already running (I have a threadsafe lock in my database for that), and if not, it creates a "normal" bullmq job, which the python service is listening to.
> 
> This, at least for my use case, mimics repeated job behavior. Maybe this helps some folks coming here.
> 
> I'm happy though, when repeated jobs arrive in python 👍 

Apscheduler does support persistent job in sql db and in memory with redis so you just use apscheduler to emit the job to bullmq when needed
